clustered/server/src/main/java/org/ehcache/clustered/server/offheap/OffHeapChainStorageEngine.java

```diff
@@ -532,14 +532,14 @@ public ReplaceResponse replace(Chain expected, Chain replacement) {
       }
     }
 
-    public ReplaceResponse removeHeader(Chain header) {
+    public boolean removeHeader(Chain header) {
```

Comment:
Can you please document. Especially the return type

Reply:
just reverting a previous change, I think it's better to create another PR to handle javadoc stuff.

Reply:
Why? Javadoc goes with the code. What does the boolean mean? Should be easy to document. Only this one.

Reply:
I think it's better to document all related methods (like replaceHeader) as well to get better understanding, not just this one.

Reply:
One is better than nothing

Reply:
added doc for removeHeader and replaceHeader